drf_user/managers.py

```diff
@@ -15,7 +15,15 @@ class UserManager(BaseUserManager):
 
     use_in_migrations = True
 
-    def _create_user(self, username, email, password, fullname, mobile, **extra_fields):
+    def _create_user(
+        self,
+        username: str,
+        email: str,
+        password: str,
+        fullname: str,
+        mobile: str = None,
+        **kwargs: dict
```

Comment:
```suggestion
        **kwargs: Dict
```
`dict` is not used in the type annotation

Reply:
Also, consider removing it as `**kwargs` is self-explanatory